lib/WhmcsAPI/Config.php

```
@@ -15,4 +15,19 @@ public static function getSystemUrl(): string
 
         return $response['value'];
     }
+
+    public static function loadModuleConfig(): array
+    {
+        $configFile = dirname(__DIR__, 2) . '/config.php';
+
+        if (is_file($configFile)) {
+            $config = require $configFile;
+
+            if (is_array($config)) {
+                return $config;
+            }
+        }
+
+        return [];
```

m.b. we should throw an exception if config does not exist?